cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/issues-list.tsx

```diff
@@ -131,45 +131,70 @@ export default function LabelsListComponent(): JSX.Element {
             </div>
             <div className='cvat-objects-sidebar-issues-list'>
                 {frameIssues.map(
-                    (frameIssue: any): JSX.Element => (
-                        <div
-                            key={frameIssue.id}
-                            id={`cvat-objects-sidebar-issue-item-${frameIssue.id}`}
-                            className={
-                                `cvat-objects-sidebar-issue-item ${frameIssue.resolved ? 'cvat-objects-sidebar-issue-resolved' : ''}`
-                            }
-                            onMouseEnter={() => {
-                                const element = window.document.getElementById(
-                                    `cvat_canvas_issue_region_${frameIssue.id}`,
-                                );
-                                if (element) {
-                                    element.setAttribute('fill', 'url(#cvat_issue_region_pattern_2)');
-                                }
-                                dispatch(activateObject(null, null, null));
-                            }}
-                            onMouseLeave={() => {
-                                const element = window.document.getElementById(
-                                    `cvat_canvas_issue_region_${frameIssue.id}`,
-                                );
-                                if (element) {
-                                    element.setAttribute('fill', 'url(#cvat_issue_region_pattern_1)');
+                    (frameIssue: any): JSX.Element => {
```

Comment:
```suggestion
                    (frameIssue: Issue): JSX.Element => {
```
Lets import Issue from cvat-core

Reply:
@klakhov Done